app/src/ui/merge-conflicts/merge-conflicts-warning.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react'
+import { Button } from '../lib/button'
+import { ButtonGroup } from '../lib/button-group'
+import { Dialog, DialogContent, DialogFooter } from '../dialog'
+import { Dispatcher } from '../../lib/dispatcher'
+import { RepositorySection } from '../../lib/app-state'
+import { Repository } from '../../models/repository'
+
+interface IMergeConflictsWarningProps {
+  readonly dispatcher: Dispatcher
+  readonly repository: Repository
+  readonly onDismissed: () => void
+}
+
+export class MergeConflictsWarning extends React.Component<
+  IMergeConflictsWarningProps,
+  {}
+> {
+  private onSubmit = () => {
+    this.props.dispatcher.changeRepositorySection(
+      this.props.repository,
+      RepositorySection.Changes
+    )
+    this.props.onDismissed()
+  }
+
+  public render() {
+    return (
+      <Dialog
+        id="merge-conflicts-warning"
+        title={__DARWIN__ ? 'Merge Conflicts Found' : 'Merge conflicts found'}
+        onDismissed={this.props.onDismissed}
+        onSubmit={this.onSubmit}
+      >
+        <DialogContent>
+          <p>Conflicts were detected as part of the last merge operation.</p>
+          <p>
+            You will need to view and resolve these conflicts before finishing
```

Comment:
should we specifically say `You will need to view and resolve these conflicts via your IDE or terminal before finishing...`? or should we be vague. I just don't want anyone to get the impression that Desktop can help them at this point

Reply:
I settled on `... via your editor or shell before finishing ...` to stick to concepts we talk about in Desktop.